pkg/controller/statefulset/stateful_set_utils.go

```diff
@@ -88,6 +91,18 @@ func getPodName(set *apps.StatefulSet, ordinal int) string {
 	return fmt.Sprintf("%s-%d", set.Name, ordinal)
 }
 
+// getClaimPodName gets the name of the Pod associated with the Claim, or an empty string if this doesn't look matching.
+func getClaimPodName(set *apps.StatefulSet, claim *v1.PersistentVolumeClaim) string {
```

Comment:
It's used in tests. It fits nicely here, next to getPodName et. al. where it's easy to see the naming convention. But since it's only used in tests maybe it's best moved to test.go?

Whatever you think is nicest.

Reply:
nit: its better to keep a test only helper in the test function